src/components/TagBrowser.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+
+import PageWrapper from './PageWrapper'
+import Navbar from './Navbar'
+import NewNoteFooter from './NewNoteFooter'
+
+const TagBrowserList = styled.main`
+  padding: 5px 8px;
+`
+
+export default class TagBrowser extends Component {
+
+  navIcons = [
+    {
+      name: 'list',
+      link: '/list'
+    },
+    {
+      name: 'plus-circle',
+      link: '/create'
+    },
+  ]
+
+  render () {
+    const { tags } = this.props
+    const tagsInAlphabeticalOrder = tags.sort((a, b) => {
+      if (a.name.toLowerCase() < b.name.toLowerCase()) { return -1; }
```

Comment:
Mach doch a und b in der Zeile davor einmal zu lowerCase, dann wird es übersichtlicher.

Reply:
Semikolon hinter der -1 kann raus.

Reply:
meinst Du so?
```
const tagsInAlphabeticalOrder = tags.sort((a, b) => {
      a = a.name.toLowerCase()
      b = b.name.toLowerCase()
      if (a < b) { return -1 }
      if (a > b) { return 1 }
      return 0;
    })
```